src/Microsoft.OData.Core/ODataBatchMimeWriter.cs

```diff
@@ -0,0 +1,564 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataBatchMimeWriter.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Core

```

Comment:
Should probably create a "Mime" namespace (and corresponding folder) and put the Mime classes there.